day_7/ceasar_cipher.rb

```diff
@@ -0,0 +1,22 @@
+def shift_char(c, base, offset)
+  (((c.ord - base) + offset) % 26 + base).chr
+end
+
+def cipher(string, offset)
+  string.chars.map do |c|
+    case c
+    when 'a'..'z'
+      shift_char(c, 'a'.ord, offset)
+    when 'A'..'Z'
+      shift_char(c, 'A'.ord, offset)
+    else
+      c
+    end
+  end.join
+end
```

Comment:
A lot of really great things happening here. Good use of iteration with `.map`. Great job using a case statement here. Good work calling another method `shift_char` from this method. Excellent!